website/content/docs/nia/compatibility.mdx

```diff
@@ -7,28 +7,26 @@ description: >-
 
 # Compatibility
 
-## Consul
-
-Below are the supported Consul versions with compatible Consul-Terraform-Sync versions. The latest Consul-Terraform-Sync binary targets supporting the latest patch version of the three most recent Consul minor versions.
-
-| Consul Version               | Compatible Consul-Terraform-Sync Version |
-| ---------------------------- | ---------------------------------------- |
-| 1.8+                         | 0.1+                                     |
-
-## Terraform
-
-Consul-Terraform-Sync is compatible with the following Terraform OSS versions:
-
-| Consul-Terraform-Sync | Compatible Terraform Version |
-| --------------------- | ---------------------------- |
-| 0.2+                  | 0.13 - 1.0                   |
-| 0.1                   | 0.13 - 0.14                  |
-
-## Terraform Cloud
-
-Consul-Terraform-Sync integration with Terraform Cloud is supported for the following:
-
-| Consul-Terraform-Sync Enterprise | Terraform Cloud            | Version            |
-| -------------------------------- | -------------------------- | ------------------ |
-| 0.4+                             | Terraform Cloud            | Latest             |
-| 0.3+                             | Terraform Enterprise       | v202010-2 - Latest |
+## Consul-Terraform-Sync
+
+Below are the supported Consul, Terraform, and Vault versions with compatible Consul-Terraform-Sync versions.
+
+|                                                                                                    | **CTS OSS**<br/><span style={{color:'#626873'}}><sub>v0.1+</sub></span> | **CTS Enterprise**<br/><span style={{color:'#626873'}}><sub>v0.1+</sub></span> |
+| :------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------: | :----------------------------------------------------------------------------: |
+|             **Consul OSS**<br/><span style={{color:'#626873'}}><sub>v1.8+</sub></span>             |             <span style={{color:'#2EB039'}}>&check;</span>              |                 <span style={{color:'#2EB039'}}>&check;</span>                 |
+|         **Consul Enterprise**<br/><span style={{color:'#626873'}}><sub>v1.8+</sub></span>          |             <span style={{color:'#2EB039'}}>&check;</span>              |                 <span style={{color:'#2EB039'}}>&check;</span>                 |
+|             **HCP Consul**<br/><span style={{color:'#626873'}}><sub>v1.8+</sub></span>             |             <span style={{color:'#2EB039'}}>&check;</span>              |                 <span style={{color:'#2EB039'}}>&check;</span>                 |
```

Comment:
Remove compatibility with HCP Consul as a row unless it has been validated by the engineering team